README.md

```diff
@@ -1,19 +1,100 @@
 # Submit a GAR Public Site UI
 
-## Docker Usage
-The following depends on a .env file in the root of this project with the required environment variables. This will start a local test instance of keycloak for local testing.
+## Requirements
+- docker
+- docker-compose
+- node
+- npm
+- your email added to GOV Notify Dev whitelist by repository administrator
+
+## Development
+
+The following depends on a `.env.dev` file in the root of this project with
+the required environment variables. To obtain it please contact repository administrator.
+
+This will start local instances of:
+- Database (used for both Frontend and Backend)
+- API (you need a local copy of `api` docker image loaded, 
+       please contact repository administrators to obtain it)
+- ClamAV engine
```

Comment:
The text is 
"You have the option to create a GAR using our online web form, or by uploading a pre-populated template **from** file. Please select how you would like to create your GAR to proceed." but it should be 
"You have the option to create a GAR using our online web form, or by uploading a pre-populated template file. Please select how you would like to create your GAR to proceed.

Reply:
Should be done